packages.dhall

```diff
@@ -0,0 +1,138 @@
+{-
```

Comment:
Would you mind removing this and the comment from the spago.dhall also? They are a lot of noise that don't add that much value IMO